src/abstractdataframe/abstractdataframe.jl

```diff
@@ -942,6 +943,80 @@ function _vcat(dfs::AbstractVector{<:AbstractDataFrame})
     return DataFrame(cols, header)
 end
 
+##############################################################################
+##
+## repeat
+##
+##############################################################################
+
+"""
+    repeat(df::AbstractDataFrame; inner::Integer = 1, outer::Integer = 1)
+
+Construct repeat each row using the `inner` argument specifies the number of
```

Comment:
"Construct repeat"? Same in the other docstring.

Below, "combination" is a strange word for this. Maybe just "data frame" or "full set of rows"?